portfolio/src/main/webapp/assets/js/main.js

```diff
@@ -70,19 +70,17 @@ jQuery(document).ready(function($) {
 
 async function getTestimonials() {
   const response = await fetch('/data');
-  const result = await response.json();
+  const arrTestimonials = await response.json();
+  console.log(arrTestimonials);
   
-  arrTestimonials = result["arrTestimonials"];
+//   arrTestimonials = result["arrTestimonials"];
 
   const testimonialsListElement = document.getElementById('testimonials');
   testimonialsListElement.innerHTML = '';
 
   for (let i = 0; i < arrTestimonials.length; i++) {
-      testimonialsListElement.appendChild(createTestimonialElement(arrTestimonials[i], "authorName", "titleName"));
+      testimonialsListElement.appendChild(createTestimonialElement(arrTestimonials[i]["text"], arrTestimonials[i]["name"], arrTestimonials[i]["title"]));
```

Comment:
Nit: You can access properties directly when they don't need to be variables.
```suggestion
      testimonialsListElement.appendChild(createTestimonialElement(arrTestimonials[i].text, arrTestimonials[i].name, arrTestimonials[i].title));
```

Reply:
is there a reason to prefer one method over the other?

Reply:
Not really, just helps with readability IMO.